lib/lighthouse.js

```diff
@@ -72,6 +72,58 @@ exports.findByPwaId = function(pwaId) {
 };
 
 /**
+ * Retrieves the Lighthouse data for a PWA
+ *
+ * @param {number} pwaId
+ * @return {Promise<Lighthouse[]>}
+ */
+exports.getLighthouseByPwaId = function(pwaId) {
+  return new Promise((resolve, reject) => {
+    // Gets the last 2 years of Ligthouses for a PWA
+    const query = ds.createQuery(ENTITY_NAME)
+      .filter('pwaId', '=', parseInt(pwaId, 10)).order('date', {descending: true}).limit(730);
+    ds.runQuery(query, (err, lighthouses) => {
+      if (err) {
+        return reject(err);
+      }
+      return resolve(lighthouses);
+    });
+  });
+};
+
+/**
+ * Retrieves the Lighthouse Grpah data for a PWA
+ * in Google Charts JSON format
+ *
+ * @param {number} pwaId
+ * @return {Promise<Json>}
+ */
+exports.getLighthouseGraphByPwaId = function(pwaId) {
+  return new Promise((resolve, reject) => {
```

Comment:
I think it should be possible to eliminate the `Promise` constructor and just `return this.getLighthouseByPwaId(…)`.